packages/rollup-tests/test/sourcemaps/index.js

```diff
@@ -33,7 +33,8 @@ runTestSuiteWithSamples(
 						};
 						const outputOptions = {
 							exports: 'auto',
-							file: directory + '/_actual/bundle.' + format + '.js',
+							dir: directory + '/dist/',
```

Comment:
to get the same result, you could use workaround in #624.

```suggestion
							entryFileName:  directory + '/_actual/bundle.' + format + '.js',
```

Reply:
The `entryFileName` hasn't add into binding. So here just using `dir`.

Reply:
Ok. If there isn't any downside, I'm ok with both.